ci.ps1

```diff
@@ -147,3 +147,23 @@ function Set-ActiveRubyFromPuppet
 
   [System.Environment]::SetEnvironmentVariable('Path', $path, [System.EnvironmentVariableTarget]::Machine)
 }
+
+# Define system profile which will print to file when loaded
+# This allows testing if profiles are loaded while executing bolt 
+function Set-Profile($profileTracker)
+{
+  Write-Host $profileTracker
+  $PsVersionTable
+  Write-Host $ENV:windir
+  $allProfiles = Join-Path -Path $ENV:WINDIR -ChildPath 'System32\WindowsPowerShell\v1.0\profile.ps1'
+  Write-host $allProfiles
+  if (!(Test-Path -Path $allProfiles))
+  {
+    New-Item -Type File -Path $allProfiles -Force
+  }
+  $BaseDir = Split-Path $profileTracker
+  New-Item -ItemType Directory -Path $BaseDir
+  $ProfileCode = [string]::Format('"Profile Loaded" | Out-File {0} -Append', $profileTracker)
```

Comment:
$Profile evaluates to null when executed from this script. Additionally setting it like this results in 
```
Successfully established WinRM connection with roddypiper after 1 attempt(s)
C:\ProfileDir\profile_tracker.txt
Name                           Value                                                                                                                                                                                                 
----                           -----                                                                                                                                                                                                 
PSVersion                      5.1.14409.1012                                                                                                                                                                                        
PSEdition                      Desktop                                                                                                                                                                                               
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                               
BuildVersion                   10.0.14409.1012                                                                                                                                                                                       
CLRVersion                     4.0.30319.42000                                                                                                                                                                                       
WSManStackVersion              3.0                                                                                                                                                                                                   
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                   
SerializationVersion           1.1.0.1                                                                                                                                                                                               
C:\windows
C:\windows\System32\WindowsPowerShell\v1.0\profile.ps1
LastWriteTime : 6/10/2019 8:10:37 PM
Length        : 0
Name          : profile.ps1
PSPath            : Microsoft.PowerShell.Core\FileSystem::C:\ProfileDir
PSParentPath      : Microsoft.PowerShell.Core\FileSystem::C:\
PSChildName       : ProfileDir
PSDrive           : C
PSProvider        : Microsoft.PowerShell.Core\FileSystem
PSIsContainer     : True
Name              : ProfileDir
FullName          : C:\ProfileDir
Parent            : 
Exists            : True
Root              : C:\
Extension         : 
CreationTime      : 6/10/2019 8:10:37 PM
CreationTimeUtc   : 6/10/2019 8:10:37 PM
LastAccessTime    : 6/10/2019 8:10:37 PM
LastAccessTimeUtc : 6/10/2019 8:10:37 PM
LastWriteTime     : 6/10/2019 8:10:37 PM
LastWriteTimeUtc  : 6/10/2019 8:10:37 PM
Attributes        : Directory
Mode              : d-----
BaseName          : ProfileDir
Target            : {}
LinkType          : 
"Profile Loaded" | Out-File C:\ProfileDir\profile_tracker.txt -Append
bundle exec rake appveyor
..........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................*........*.........................................................................................................................................................................C:\ProfileDir\profile_tracker.txt
No such file or directory @ rb_sysopen - C:\windows\System32\WindowsPowerShell\v1.0\profile.ps1
.
.......................................................................................................................
Pending: (Failures listed here are expected and do not affect your suite's status)
```
Which makes it seem like the file is not persisted (or at least connot be read in rspec context). Either way it does not seem possible to create a profile that will get loaded with this approach. I think we may been do do this in acceptance tests.